app/models/performance_statistics.rb

```diff
@@ -30,7 +29,7 @@ def candidate_query
               END status
           FROM
               application_forms f
-          FULL OUTER JOIN
+          LEFT JOIN
```

Comment:
Right — so this ignores candidates without forms?

Reply:
Yes, it just gives a list of forms joined to their candidate